### PR TITLE
fix: small tailwind overwrite issues

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- We had a few overwrites with tailwind upgrade that showed up

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/modals/common/Modal.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ export const modalSizeToClassName: Record<ModalSize, string> = {
   [ModalSize.XSmall]: 'w-[21.25rem]',
   [ModalSize.Small]: 'w-[26.25rem]',
   [ModalSize.Medium]: 'w-[35rem]',
-  [ModalSize.Large]: 'w-[42.5rem]',
+  [ModalSize.Large]: '!w-[42.5rem]',
```
Comment (rebelchris) [inline packages/shared/src/components/modals/common/Modal.tsx:73]: Unfortunately we use this for the actual page, which also has a w-full so always get's overruled.

Comment (sshanzel) [reply packages/shared/src/components/modals/common/Modal.tsx:73]: Oh. So the tailwind update prioritizes pre-defined values or arbitrary values now?

File: packages/shared/tailwind.config.js [hunk 2]
```diff
@@ -137,6 +137,7 @@ module.exports = {
       2: '2',
       3: '3',
       rank: '3',
+      header: '4',
```
Comment (sshanzel) [inline packages/shared/tailwind.config.js:140]: Nice, thanks for adding this.

**Review by sshanzel — APPROVED** [review]
Looks great to me! Thank you for the fix! 🚀

**Pull request merged**